libs/motr/motr_core_k8s_lib.py

```diff
@@ -140,30 +142,25 @@ def get_cortx_node_endpoints(self, cortx_node=None):
                 return self.node_dict[key]
         return None
 
-    def get_number_of_m0clients(self, cluster_info_dic=None):
+    def get_number_of_motr_clients(self):
         """
-        To get the number of m0clients in a node
-
-        :param cluster_info_dic- Dictionray containing cluster info
-        :type: dictionary
-        :returns: Number of m0clients present in given node
+        To get the number of motr_clients in a node
+        :returns: Number of motr_clients present in given node
         :rtype: integer
         """
-        if cluster_info_dic is None:
-            return len(self.node_dict[self.get_primary_cortx_node()]["m0client"])
-        return len(cluster_info_dic[self.get_primary_cortx_node()]["m0client"])
+        return len(self.node_dict[list(self.node_pod_dict.keys())[0]]["motr_client"])
 
-    def get_node_name_from_pod_name(self, data_pod=None):
+    def get_node_name_from_pod_name(self, motr_client_pod=None):
```

Comment:
default value of motr_client _pod should not be none here as its mandatory parameter.